run_dir/design/ont_flowcell.html

```diff
@@ -0,0 +1,153 @@
+              <dl class="dl-horizontal" style="margin-bottom: 0px;">
+            {% elif args[e] == "sub_pair" %}
+              <dt style="text-align: left; margin-left: 10px; font-weight: normal; width: 210px;">{{ e[0] }}</dt>
+              <dd>{{ e[1] }}</dd>
+            {% end %}
+
+          {% end %}
+        </dl>
+      </div>
+    </div>
+
+    <div class="tabbable mb-3">
+      <ul class="nav nav-tabs">
+        <li class="nav-item">
+          <a class="nav-link active" href="#tab_barcode_information" role="tab" data-toggle="tab">Barcode Information</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#tab_running_notes_content" role="tab" data-toggle="tab">Running Notes</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#tab_links_content" role="tab" data-toggle="tab">Links</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/flowcells/{{ flowcell.get('name') }}/report" target="_blank">MinKNOW run report</a>
+        </li>
+      </ul>
+    </div>
+    <div class="tab-content">
+      <div class="tab-pane fade show active" id="tab_barcode_information">
+        <table class="table table-bordered narrow-headers no-margin right_align_headers" id="ont_barcodes">
+            <thead id="ont_fc_table_head">
+                <tr class="sticky darkth">
+                    <th>Barcode</th>
+                    <th>Alias</th>
+                    <th class="text-right">Read count passed</th>
+                    <th>% total</th>
+                    <th class="text-right">Yield passed (bp)</th>
+                    <th>% total</th>
+                    <th class="text-right" data-toggle="tooltip" title="Obtained by dividing yield by read count">Avg. length passed (bp)</th>
+                    <th class="text-right">% Pass / Fail</th>
+                </tr>
+            </thead>
+            <tbody class="list">
+            {% if barcodes %}
+              {% for bc in barcodes %}
+              <tr>
+                  <td>{{ barcodes[bc]["bc_name"] }}</td>
+                  <td>{{ barcodes[bc]["barcode_alias"] }}</td>
+                  <td class="text-right">{{ '{:,}'.format(barcodes[bc]["basecalled_pass_read_count"]) }}</td>
+                  <td>{{ barcodes[bc]["basecalled_pass_read_count_pc"] }}</td>
+                  <td class="text-right">{{ '{:,}'.format(barcodes[bc]["basecalled_pass_bases"]) }}</td>
+                  <td>{{ barcodes[bc]["basecalled_pass_bases_pc"] }}</td>
+                  <td class="text-right">{{ '{:,}'.format(barcodes[bc]["average_read_length_passed"]) }}</td>
+                  <td class="text-right">{{ barcodes[bc]["accuracy"] }}</td>
+              </tr>
+              {% end %} 
+            {% end %}
+            </tbody>
+        </table>
+      </div>
+      <!-- FLOWCELL NOTES TAB -->
+      {% include "running_notes_tab.html" %}
```

Comment:
Running notes can be fixed in the next PR.

Reply:
b97f4401fdd8cc792217257a8e3328041e42f222